hummingbot/templates/conf_amm_arb_strategy_TEMPLATE.yml

```diff
@@ -33,3 +33,7 @@ market_2_slippage_buffer: null
 # If false, the bot will wait for first exchange order filled before submitting the other order
 concurrent_orders_submission: null
 
+# Whether to use rate oracle on unmatched trading pairs
+# Set this to either True or False
+use_oracle_conversion_rate: null
+
```

Comment:
Should we increment the version on https://github.com/CoinAlpha/hummingbot/blob/2c7403db8c7d6574bcce0b0f5ad350db11add633/hummingbot/templates/conf_amm_arb_strategy_TEMPLATE.yml#L5 when adding new parameter on the template?

Reply:
That is a good point. I have committed the change.